test/cmd/images-old-policy.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/../..
+source "${OS_ROOT}/hack/lib/init.sh"
+os::log::stacktrace::install
+trap os::test::junit::reconcile_output EXIT
+
+# Cleanup cluster resources created by this test
+(
+  set +e
+  oc login -u system:admin
+  oadm policy reconcile-cluster-roles --additive-only=false --confirm
+  exit 0
+) &>/dev/null
+
+
+os::test::junit::declare_suite_start "cmd/images-old-policy"
+
+oc project -q
+project=$(oc project -q)
```

Comment:
We do "cleanup" steps with reconciliation of roles without logging in as admin in other tests --why is the login necessary here? 

We can do two things to mitigate *this* issue:
 - poll until the project cache is up to date with the new project we create before launching a test bucket
 - log back in to the correct user/context at the end of your cleanup step if `system:admin` is necessary